hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/MonoTransactionDispatcher.java

```diff
@@ -115,6 +116,20 @@ private void finishConsensusCreateTopic(@NonNull final HandleContext handleConte
         topicStore.commit();
     }
 
+    private void dispatchFreeze(@NonNull final HandleContext handleContext) {
+        requireNonNull(handleContext);
+        final var handler = handlers.freezeHandler();
+        handler.handle(handleContext);
+        finishFreeze(handleContext);
+    }
+
+    private void finishFreeze(@NonNull final HandleContext handleContext) {
```

Comment:
If we add this empty `finishFreeze()`, we should make it `protected` so a subclass could use it for other purposes.

Reply:
I only put it in there because all of the other dispatchX methods call a finishX method. All of the finishX methods are private, so I think that finishFreeze() should be too. Arguably, maybe it would be preferred to delete this finishFreeze method because it doesn't need to exist.